equinox/nn/attention.py

```diff
@@ -160,6 +300,7 @@ def __init__(
         self.output_proj = Linear(
             num_heads * vo_size, output_size, use_bias=use_output_bias, key=okey
         )
+
```

Comment:
Not that this change in and of itself matters, but can spurious changes be kept to a minimum?